modules/supported-platforms-for-openshift-clusters.adoc

```diff
@@ -9,42 +9,44 @@
 
 In {product-title} {product-version}, you can install a cluster that uses installer-provisioned infrastructure on the following platforms:
 
+* Alibaba Cloud
 * Amazon Web Services (AWS)
+* Bare metal
 * Google Cloud Platform (GCP)
+* IBM Cloud&#174 VPC
 * Microsoft Azure
 * Microsoft Azure Stack Hub
+* Nutanix
 * {rh-openstack-first}
 ** The latest {product-title} release supports both the latest {rh-openstack} long-life release and intermediate release. For complete {rh-openstack} release compatibility, see the link:https://access.redhat.com/articles/4679401[{product-title} on {rh-openstack} support matrix].
-* IBM Cloud VPC
-* Nutanix
 * VMware vSphere
-* Alibaba Cloud
-* Bare metal
 
 For these clusters, all machines, including the computer that you run the installation process on, must have direct internet access to pull images for platform containers and provide telemetry data to Red Hat.
 
 [IMPORTANT]
 ====
 After installation, the following changes are not supported:
 
-* Mixing cloud provider platforms
-* Mixing cloud provider components, such as using a persistent storage framework from a differing platform than what the cluster is installed on
+* Mixing cloud provider platforms.
+* Mixing cloud provider components. For example, using a persistent storage framework from a another platform on the platform where you installed the cluster. 
 ====
 
 In {product-title} {product-version}, you can install a cluster that uses user-provisioned infrastructure on the following platforms:
 
 * AWS
 * Azure
 * Azure Stack Hub
+* Bare metal
 * GCP
+* {ibmpowerProductName}
+* {ibmzProductName} or {linuxoneProductName}
 * {rh-openstack}
 ** The latest {product-title} release supports both the latest {rh-openstack} long-life release and intermediate release. For complete {rh-openstack} release compatibility, see the link:https://access.redhat.com/articles/4679401[{product-title} on {rh-openstack} support matrix].
-* VMware vSphere
 * VMware Cloud on AWS
-* Bare metal
-* {ibmzProductName} or {linuxoneProductName}
-* {ibmpowerProductName}
+* VMware vSphere
+
+Depending on the supported cases for the platform, you can perform installations on user-provisioned infrastructure, so that you can run machines with full internet access, place your cluster behind a proxy, or perform a restricted network installation. 
```

Comment:
```suggestion
Depending on the supported cases for the platform, you can perform installations on user-provisioned infrastructure, so that you can run machines with full internet access, place your cluster behind a proxy, or perform a disconnected installation. 
```